saleor/settings.py

```diff
@@ -113,6 +113,9 @@ def get_url_from_env(name, *, schemes=None) -> Optional[str]:
         conn_max_age=DB_CONN_MAX_AGE,
     ),
 }
+DATABASES[DATABASE_CONNECTION_REPLICA_NAME]["TEST"] = {
+    "MIRROR": DATABASE_CONNECTION_DEFAULT_NAME
+}
```

Comment:
Should it be in `saleor/tests/settings.py`?

Reply:
In Django docs this setting is part of the main DATABASES setting. For me the `test/settings.py` is more a place to override the defaults from the main `settings.py`, but I won't argue about this, both places should be OK.